### PR TITLE
docs: recipes and examples page: examples, deps, and tests links

### PR DESCRIPTION
**Pull request by tcrowe**
## What does it do?

Originally I set out to fix the URL for "Recipes & Examples" at https://mockyeah.js.org/ but you already corrected it. I added a page reviewing what we were discussing in [this issue](https://github.com/mockyeah/mockyeah/issues/484).

## How to test

`npm start` from the project root.

---

Do you want this? Is there anything else to add?

### CODE REVIEW

**Comment by AndersDJohnson** [comment]
@tcrowe Yes, you've discovered the new documentation site, which we haven't published yet. It's meant to launch with Mockyeah version 1.0 which is still in alpha.

I'll review these changes and get them in - thanks for the contribution!

**Comment by tcrowe** [comment]
You're welcome. If you need changes or you want to take a different approach you wont hurt my feelings. 👍

File: packages/mockyeah-docs/src/pages/Recipes-and-Examples.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Recipes & Examples
+---
+
+There are some examples includes in the [mockyeah monorepo examples directory](https://github.com/mockyeah/mockyeah/tree/master/packages/mockyeah/examples). GitHub tracks how [users depend on the mockyeah package](https://github.com/mockyeah/mockyeah/network/dependents?package_id=UGFja2FnZS0xNDkzNzMwNA%3D%3D).
```
Comment (AndersDJohnson) [inline packages/mockyeah-docs/src/pages/Recipes-and-Examples.md]:
```suggestion
There are some examples included in the [mockyeah monorepo examples directory](https://github.com/mockyeah/mockyeah/tree/master/packages/mockyeah/examples). GitHub tracks how [users depend on the mockyeah package](https://github.com/mockyeah/mockyeah/network/dependents?package_id=UGFja2FnZS0xNDkzNzMwNA%3D%3D).
```

File: packages/mockyeah-docs/src/pages/Recipes-and-Examples.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+[Test Suites](https://github.com/mockyeah/mockyeah/tree/master/packages/mockyeah/test/mockyeah) connect to [Fixtures](https://github.com/mockyeah/mockyeah/tree/master/packages/mockyeah/test/fixtures) in the unit tests.
```
Comment (AndersDJohnson) [inline packages/mockyeah-docs/src/pages/Recipes-and-Examples.md]:
```suggestion
Mock suites at [packages/mockyeah/test/mockyeah](https://github.com/mockyeah/mockyeah/tree/master/packages/mockyeah/test/mockyeah) connect to fixtures at [packages/mockyeah/test/fixtures](https://github.com/mockyeah/mockyeah/tree/master/packages/mockyeah/test/fixtures) in the unit tests.
```

**Pull request merged**